plugin/aws/ec2InstanceConsoleOutput.go

```diff
@@ -50,16 +50,16 @@ type consoleOutput struct {
 	content []byte
 }
 
-func (cl *ec2InstanceConsoleOutput) cachedConsoleOutput(ctx context.Context) (consoleOutput, error) {
-	output, err := plugin.CachedOp(ctx, "ConsoleOutput", cl, 30*time.Second, func() (interface{}, error) {
+func (inst *ec2Instance) cachedConsoleOutput(ctx context.Context) (consoleOutput, error) {
```

Comment:
I'd suggest moving this to `ec2Instance.go`, since that covers functions related to that type.

Reply:
Done